runbotics-orchestrator/src/main/java/com/runbotics/security/FeatureKeyConstants.java

```diff
@@ -69,6 +69,7 @@ public final class FeatureKeyConstants {
     public static final String PROCESS_COLLECTION_ADD = "PROCESS_COLLECTION_ADD";
     public static final String PROCESS_COLLECTION_EDIT = "PROCESS_COLLECTION_EDIT";
     public static final String PROCESS_COLLECTION_DELETE = "PROCESS_COLLECTION_DELETE";
+    public static final String PROCESS_COLLECTION_EVERY = "PROCESS_COLLECTION_EVERY";
```

Comment:
is it for reading or reading/adding/editing etc.?

Reply:
what do you think about COLLECTION_ALL_ACCESS or COLLECTION_MANAGE_ALL?

Reply:
I think ALL_ACCESS will be good